### ISSUE TITLE
Write additional chapter(s)

### ISSUE THREAD

**Issue by alanverdugo** [alanverdugo/gilbert#7]
In the application, there is a section for "lectures" which are just text that can be read offline.

A "chapter" is just a `.rst` document in plain English with some sort of lecture about electricity and magnetism.

Right now there are just 3 chapters and more content would be nice.

Please refer to the following directory to see the current chapters and get an idea of content, length and format: https://github.com/alanverdugo/gilbert/tree/master/assets/chapters/english

Minimum length should be 300 words.

Figures are optional but appreciated.

_**Please only submit original content that is free to use (not under any copyright).**_ 

Note: All content will be released under the Apache 2 license as part of the application itself.

**Comment by neil007m** [comment]
@alanverdugo I'd love to contribute, please feel free to assign it to me.

**Comment by alanverdugo** [comment]
Thanks @neil007m !! I've assigned it to you.
Note: I may open more issues similar to this to add even more content.

**Comment by neil007m** [comment]
@alanverdugo I got sidetracked today, should be able to get to this in under 24 hours!

**Comment by neil007m** [comment]
@alanverdugo Sounds good

> Thanks @neil007m !! I've assigned it to you.
> Note: I may open more issues similar to this to add even more content.

Sure! I got a chance to play around with the rst format and discover quite a few new things in the process. In case anyone finds it helpful in the future, [pandoc](https://pandoc.org) is a nice tool to convert a ton of formats to rst, might be useful for large documents.

**Comment by alanverdugo** [comment]
Thanks a lot for your help @neil007m !!! I've approved the pull request with your change, merged it, and I've labeled it "hacktoberfest-accepted" in case you are interested in the swag (or tree).

I will close this issue now. Thanks again!! :)

**Comment by neil007m** [comment]
@alanverdugo My pleasure! I'm definitely getting well versed with open source, this was a good exercise.

**Comment by alanverdugo** [comment]
> @alanverdugo My pleasure! I'm definitely getting well versed with open source, this was a good exercise.

Hello @neil007m , sorry I didn't ask this before, but I am about to create a new release and would like to know how do you want to be mentioned in the "collaborators" section. Should I just use "neil007m"? your real name? anything else?

Please let me know, thanks!

**Comment by neil007m** [comment]
Hey @alanverdugo - something like @neil007m (Neil Matin) would be perfect, thanks!
